docs/documentation/functions/supporting/Test-PrereqDnsEntries.md

```diff
@@ -0,0 +1,64 @@
+# Test-PrereqDnsEntries
+
+## Synopsis
+
+Checks the DNS resolution of a DNS entry or entries on a DNS server or servers.
+
+## Syntax
+
+```powershell
+Test-PrereqDnsEntries [-dnsEntries] <String[]> [-dnsServers] <String[]> [<CommonParameters>]
+```
+
+## Description
+
+The `Test-PrereqDnsEntries` cmdlet checks for the resolution of a DNS record or multiple DNS records and tests the forward and reverse lookups in a DNS server or multiple DNS servers.
+
+## Examples
+
+### Example 1
+
+```powershell
+Test-PrereqDnsEntries -dnsEntries sfo-vcf01.sfo.rainpole.io, 192.168.11.41 -dnsServers rpl-ad01.rainpole.io
+This example will validate the forward and reverse details of the two DNS entries and if one exists in the DNS Server and return the cooresponding value.

```

Comment:
```suggestion
This example will validate the forward and reverse details of the two DNS entries and if one exists in the DNS Server it will return the corresponding value.

```

Reply:
Updated In https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/839b36abdacf09ae8380c818b8ebaab40f61f34a